sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyAsyncClient.java

```diff
@@ -1366,5 +1367,62 @@ Duration getDefaultPollingInterval() {
             return monoError(logger, ex);
         }
     }
+
+    /**
+     * Get the requested number of bytes containing random values from a managed HSM.
+     *
+     * <p><strong>Code Samples</strong></p>
+     * <p>Gets a number of bytes containing random values from a Managed HSM. Prints out the retrieved bytes in
+     * base64Url format.</p>
+     *
+     * {@codesnippet com.azure.security.keyvault.keys.KeyAsyncClient.getRandomBytes#int}
+     *
+     * @param amount The requested number of random bytes.
+     *
+     * @return The requested number of bytes containing random values from a managed HSM.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<RandomBytes> getRandomBytes(int amount) {
```

Comment:
Unless it's idiomatic, let's stick with `count` like in the swagger. These terms were discussed at length.